src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -3790,7 +3790,10 @@ public function set_modifiable_text( string $plaintext_content ): bool {
 				 * properly escape these things, but this could mask regex patterns
 				 * that previously worked. Resolve this by not sending `</script`
 				 */
-				if ( false !== stripos( $plaintext_content, '</script' ) ) {
+				if (
+					false !== stripos( $plaintext_content, '</script' ) ||
+					false !== stripos( $plaintext_content, '<script' )
```

Comment:
A possible nano performance improvement and a bit easier to read:

```suggestion
				$lowercase_plaintext_content = strtolower( $plaintext_content );
				if (
					str_contains( $lowercase_plaintext_content, '</script' ) ||
					str_contains( $lowercase_plaintext_content, '<script' )
```

Reply:
how is this a nano performance improvement? It seems like it would be nano-heavier by allocating a copy of the string and lowercasing it. `stripos` can compare ASCII-case-insensitively without doing that.

readability aside, which seems subjective.